main/real_em.F

```diff
@@ -118,13 +119,19 @@ END SUBROUTINE Setup_Timekeeping
    !  The configuration switches mostly come from the NAMELIST input.
 
 #ifdef DM_PARALLEL
+   CALL wrf_get_dm_communicator( save_comm )
+   CALL wrf_set_dm_communicator( mpi_comm_allcompute )
    IF ( wrf_dm_on_monitor() ) THEN
       CALL initial_config
    END IF
    CALL get_config_as_buffer( configbuf, configbuflen, nbytes )
    CALL wrf_dm_bcast_bytes( configbuf, nbytes )
    CALL set_config_as_buffer( configbuf, configbuflen )
    CALL wrf_dm_initialize
+#ifndef STUBMPI
+   CALL MPI_BARRIER( mpi_comm_allcompute, ierr )
```

Comment:
@honnorat 
Marc,
We do not have very many bare MPI calls in the part of the code that users work in, for example in the main/real_em.F file. What we do is wrap the MPI calls, and put that wrapper in the externals/RSL_LITE/module_dm.F file. For example, calling it wrf_dm_mpi_barrier, would be conventional. In the module_dm.F file, that is where the additional `ifndef STUBMPI` could be located.

Reply:
Yes indeed. Since reworking MPI calls in WRF would go beyond the time I can devote to this task, I simply moved the barrier in `module_dm.F`. Is this appropriate?